build.sbt

```diff
@@ -155,3 +155,5 @@ lazy val `clio-integration-test` = project
   .disablePlugins(AssemblyPlugin)
   .settings(commonSettings)
   .settings(libraryDependencies ++= Dependencies.IntegrationTestDependencies)
+
+libraryDependencies += "com.google.cloud" % "google-cloud-storage" % "1.14.0"
```

Comment:
What error were you seeing when you added this through the normal way?

Reply:
the library wasn't visible to my code, e.g. import didn't work